README.md

```
@@ -239,36 +155,343 @@
+`session.wait()` to block execution until you close the App. See
+[this page](https://voxel51.com/docs/fiftyone/user_guide/app.html#creating-a-session)
+for more information.
+
+
+![------------------------------------------------------------------](https://github.com/user-attachments/assets/fb0573d0-bb56-40ff-9ae1-a5e8f62f5f42)
+
+## <img src="https://user-images.githubusercontent.com/25985824/106288517-2422e000-6216-11eb-871d-26ad2e7b1e59.png" height="20px"> **key features** 🔑
+
+- **[Visualize Complex Datasets:](https://docs.voxel51.com/user_guide/app.html)** Easily explore images, videos, and associated labels in a powerful visual interface.
+
+https://github.com/user-attachments/assets/af8cd626-57b7-4f2a-96bf-1c8a513c2e2b
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)